### DISCUSSION TITLE
CodeSandbox examples not working?

### DISCUSSION THREAD

**Discussion by arvigeus** [remix-run/remix#4015] [General]
Is this just me?

I tried the [Basic](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/basic) example and I got nothing - both FF and Chrome. Something broken, or my machine acting up?

**Comment by cliffordfajardo** [comment]
I recommend checking out the the Remix template made by @kiliman . Kiliman's template uses a nodejs template, which I think the starter template is lacking.

![CleanShot 2022-08-18 at 07 15 45@2x](https://user-images.githubusercontent.com/6743796/185417565-bb8bf967-b021-47eb-ba88-6a6b813be821.png)


Here is a link to the template, feel free to fork:
https://codesandbox.io/s/optimistic-swanson-73i5dq

When creating new templates though, I recommend using the template shown in this image though; this ensure's that you're using an update to date version of Remix

**Reply by arvigeus** [reply]
It works, awesome! I even managed to make my own template with TailwindCSS and DaisyUI. Not sure how to mark this as an answer though...
